modules/using-aws-cli-create-iam-role-alb-controller.adoc

```diff
@@ -57,24 +57,25 @@ STATEMENT	sts:AssumeRoleWithWebIdentity	Allow
 STRINGEQUALS	system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster
 PRINCIPAL	arn:aws:iam:777777777777:oidc-provider/<oidc-provider-id>
 ----
-<1> Note the ARN of the created IAM role.
+<1> Note the ARN of an AWS IAM role.
 
-. Download the permission policy for the `AWSLoadBalancerController` by running the following command:
+. Download the permission policy for the AWS Load Balancer Controller by running the following command:
 +
 [source,terminal]
 ----
 $ curl -o albo-controller-permission-policy.json https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/assets/iam-policy.json
 ----
 
-. Attach the permission policy for the `AWSLoadBalancerController` to the IAM role by running the following command:
+. Attach the permission policy for the AWS Load Balancer Controller to an AWS IAM role by running the following command:
 +
 [source,terminal]
 ----
 $ aws iam put-role-policy --role-name albo-controller --policy-name perms-policy-albo-controller --policy-document file://albo-controller-permission-policy.json
 ----
 
-. Create an `AWSLoadBalancerController` resource file named `example-sts-iam-role.yaml` with contents such as the following example:
+. Create a YAML file that defines the `AWSLoadBalancerController` object:
 +
+Example `sample-aws-lb-manual-creds.yaml` file
```

Comment:
```suggestion
Example `sample-aws-lb-manual-creds.yaml` file:
```

Reply:
@opayne1 @xenolinux I don't think we need a colon here, right? This is a heading.